src/test/java/it/geosolutions/httpproxy/HttpProxyIntegrationTests.java

```diff
@@ -206,6 +207,27 @@ public void testPUTUsingWireMock() throws IOException {
         }
     }
 
+    /**
+     * Validates that HTTP URL pot set to 80 if no port is specified
+     */
+    @Test
+    public void testHttpDefaultPort() throws IOException {
+        String urlStr = "http://localhost:" + "/geostore/users/5";
+        URL url = Utils.buildURL(urlStr);
+        Assert.assertEquals(url.getPort(), Utils.DEFAULT_HTTP_PORT);
+    }
+
+    /**
+     * Validates that HTTP URL pot set to 80 if no port is specified
+     */
+    @Test
+    public void testHttpsDefaultPort() throws IOException {
+        String urlStr = "https://localhost:" + "/geostore/users/5";
```

Comment:
this can be a unique string no needs to concatenate. Also the two dots after the localhost domain should not be needed

Reply:
done